src/lib.rs

```diff
@@ -511,7 +512,12 @@ pub fn seek_info(file_size: usize, pos: usize, len: usize) -> SeekInfo {
 
 /// Returns the chunk index range [start, end) that is overlapped by the byte range defined by `pos`
 /// and `len`. Returns empty range if `file_size` is so small that there are no chunks.
-fn overlapped_chunks(file_size: usize, pos: usize, len: usize) -> (usize, usize) {
+fn overlapped_chunks(
+    file_size: usize,
+    pos: usize,
+    len: usize,
+    max_chunk_size: usize,
+) -> (usize, usize) {
     // FIX THIS SHOULD NOT BE ALLOWED
     if file_size < (3 * MIN_CHUNK_SIZE) || pos >= file_size || len == 0 {
```

Comment:
there shall no longer be a const of `MIN_CHUNK_SIZE` in production code anymore ?